content/docs/1.4.0/references/settings.md

```diff
@@ -552,10 +548,14 @@ See [Trim Filesystem](../../volumes-and-nodes/trim-filesystem) for details.
 
 > Default:
 
-This deprecated setting is replaced by 2 new settings "Guaranteed Engine Manager CPU" and "Guaranteed Replica Manager CPU" since Longhorn version v1.1.1.
+This deprecated setting is replaced by 2 new settings `Guaranteed Engine Manager CPU` and `Guaranteed Replica Manager CPU` since Longhorn version v1.1.1.
 
 #### Disable Replica Rebuild
 
 > Default: `false`
 
-This deprecated setting is replaced by the new setting 'Concurrent Replica Rebuild Per Node Limit'. Once the new setting value is 0, it means rebuilding disable.
+This deprecated setting is replaced by the new setting `Concurrent Replica Rebuild Per Node Limit`. Once the new setting value is 0, it means rebuilding disable.
+
+#### Custom mkfs.ext4 parameters
+
+This deprecated setting is replace by the new setting `mkfsParams` in the StorageClass and planned removed from v1.5.0. 
```

Comment:
Please add the original setting description before this statement. 


> Allows setting additional filesystem creation parameters for ext4. For older host kernels it might be necessary to disable the optional ext4 metadata_csum feature by specifying `-O ^64bit,^metadata_csum`. 
> 
> This deprecated setting is replaced by the new parameter `mkfsParams` in the StorageClass and is planned removed from v1.5.0.

Please make "`mkfsParams` in the StorageClass" refer to the corresponding page.